src/appengine/handlers/cron/backup.py

```diff
@@ -38,17 +39,17 @@ def _datastore_client():
   return googleapiclient.discovery.build('datastore', 'v1')
 
 
-class Handler(base_handler.Handler):
+class Handler(base_handler_flask.Handler):
   """Handler for triggering the backup URL."""
 
-  @handler.check_cron()
+  @handler_flask.check_cron()
   def get(self):
     """Handle a cron job."""
     backup_bucket = local_config.Config(
         local_config.PROJECT_PATH).get('backup.bucket')
     if not backup_bucket:
       logs.log('No backup bucket is set, skipping.')
-      return
+      return 'OK'
```

Comment:
This is coming up a lot. I think we should make the decorator for cron handlers return 'OK' by default (and be overrideable by actual return values). We can rename `check_cron` to `cron` as well to make the name more general.

Reply:
yes, nice approach. Will update the PR.